torch/csrc/jit/ir/scope.h

```diff
@@ -50,6 +51,26 @@ struct TORCH_API Scope : public c10::intrusive_ptr_target {
 
 struct Function;
 struct InlinedCallStack;
+struct ModuleInstanceInfo {
```

Comment:
Nit: It'd be good to have some comments for the class and its public methods.

Reply:
Address the comment in latest commit (d6403b92e3ceb7708c3264e1b9875163d2a2658e).